### PR TITLE
Targets refactoring

### PR DESCRIPTION
**Pull request by tmat**
Infrastructure only change.

Refactors `Imports.targets` to get them closer to similar targets in RepoToolset.

### CODE REVIEW

**Comment by tmat** [comment]
test this please

**Comment by tmat** [comment]
test windows_debug_spanish_unit32_prtest  please

**Comment by tmat** [comment]
test windows_debug_vs-integration_prtest please

**Comment by tmat** [comment]
test windows_release_vs-integration_prtest please

**Comment by tmat** [comment]
@jaredpar @dotnet/roslyn-infrastructure  Please review.

File: build/Targets/Roslyn.Toolsets.Xunit.targets [hunk 1]
```diff
@@ -18,7 +18,6 @@
     <XUnitWpfArguments>"$(OutDir)\$(AssemblyName).dll"</XUnitWpfArguments>
 
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <ForceGenerationOfBindingRedirects>true</ForceGenerationOfBindingRedirects>
```
Comment (jaredpar) [inline build/Targets/Roslyn.Toolsets.Xunit.targets]: Is this the default now?

Comment (tmat) [reply build/Targets/Roslyn.Toolsets.Xunit.targets]: Yes, `AutoGenerateBindingRedirects` just works.

**Pull request merged**